elastic/kibana
# [Maps] Support by value saved objects.

This PR updates map embeddable and map application to support saved objects by value, allowing users to store map saved objects directly in dashboard without needed to create and maintain a separate saved object.

Testing instructions
1) Add `dashboard.allowByValueEmbeddables: true` in kibana.dev.yml
2) Create new dashboard
3) Use `create new` to add a panel and select maps.
4) Create your map and save with `Save return`
5) save your dashboard. Inspect dashboard saved object. Notice how map saved object is stored directly in dashboard.
